### ISSUE TITLE
Fix Makefile error during compilation

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1520]
**Reported by Jano on 2 Nov 2014 19:50 UTC**

Fix manpage/Makefile error

```
install: impossibile eseguire stat di "airdriver-ng.8": File o directory non esistente
make[*** [install](1]:) Errore 1
make[uscita dalla directory "/home/jano/aircrack-ng/manpages"
make: *** [install](1]:) Errore 2
```
- Patch attacked.

Jano

**Comment by aircrack-ng** [comment]
**Attachment by Jano on 2 Nov 2014 19:52 UTC**

Uploaded file [makefile_airdriver-ng.patch](https://github.com/aircrack-ng/trac-attachments/blob/master/1520/20141102-195257_710-makefile_airdriver-ng.patch)

fix manpage/makefile error

**Comment by aircrack-ng** [comment]
**Attachment by Jano on 2 Nov 2014 19:53 UTC**

Uploaded file [makefile_airdriver-ng.2.patch](https://github.com/aircrack-ng/trac-attachments/blob/master/1520/20141102-195322_578-makefile_airdriver-ng.2.patch)

fix manpage/makefile error

**Comment by aircrack-ng** [comment]
**Comment by misterx on 2 Nov 2014 19:50 UTC**

Fixed in aircrack-ng/aircrack-ng@4853dc3916a1abdedd4a788e7ff88e3362b48dee. Thank you.
